pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -1136,6 +1136,26 @@ derived from RuntimeException or a checked exception.
 <![CDATA[
 public void foo() throws Exception {
 }
+]]>
+        </example>
+    </rule>
+	
+	<rule name="SignatureDeclareThrowsRuntimeException"
+          since="6.13.0"
+          message="A method/constructor should not explicitly declare java.lang.RuntimeException or it's subclasses in throws clause of it's signature"
+          class="net.sourceforge.pmd.lang.java.rule.design.SignatureDeclareThrowsRuntimeExceptionRule"
+          typeResolution="true"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_design.html#signaturedeclarethrowsruntimeexception">
+        <description>
+A method/constructor should not explicitly declare java.lang.RuntimeException or it's subclasses in throws clause of it's signature, since it is advised to avoid declaring unchecked exceptions in method signature.
+        </description>
+        <priority>3</priority>
+        <example>
+<![CDATA[
+import java.lang.RuntimeException;
```

Comment:
You do not need to import classes from package `java.lang`. They are automatically imported. PMD actually has a rule to avoid this: [DontImportJavaLang](https://pmd.github.io/pmd-6.12.0/pmd_rules_java_codestyle.html#dontimportjavalang)